doc/recognition.xml

```diff
@@ -356,8 +356,17 @@ set this to <K>false</K>. The result is either <K>fail</K> or
 <K>false</K> if the method fails or a recognition info record <C>ri</C>.
 If the method created a leaf then <C>ri</C> will be a leaf, otherwise
 it will have the attribute <Ref Attr="Homom"/> set, but no factor or
-kernel have been created or recognised yet. You can use for example
-the methods in <Ref Var="FindHomMethodsPerm"/> or
+kernel have been created or recognised yet.
+
+<P/>GAP homomorphisms are not required to give a sensible answer when
+given a value not in their source, and in practice often enter the break
+loop, or return an incorrect answer. This causes problems when checking
+if a value is not in the represented group. To avoid this problem,
+<Ref Attr="ValidateHomomInput"/> can be assigned to a function.
+This function is used to filter possible group elements <C>x</C>,
```

Comment:
Please remove `<C>x</C>`, there's no point to define a variable that's never used 🙂